app/person-escort-record/app/confirm/controllers.test.js

```diff
@@ -0,0 +1,7 @@
+// const { HandoverController } = require('./controllers')
+
+// const controller = new HandoverController({ route: '/' })
+
+describe('Person Escort Record controllers', function () {
+  describe('HandoverController', function () {})
```

Comment:
Seems legit :joy:

Reply:
As mentioned. This was to get initial feedback on code structure, not to have all supporting tests. I find this a faster approach if there is feedback, otherwise you spend more time re-working tests with each piece of feedback.